admin/acceptance.py

```diff
@@ -146,7 +147,7 @@ def start_nodes(self):
         }
         metadata.update(self.metadata)
 
-        for index in range(2):
+        for index in range(1):
```

Comment:
I will run this and check but it seems that this is changing from creating two acceptance testing nodes to one.